iOS_SDK/OneSignalSDK/Source/OneSignal.m

```diff
@@ -2243,15 +2243,14 @@ + (BOOL)remoteSilentNotification:(UIApplication*)application UserInfo:(NSDiction
         }
     }
     // Method was called due to a tap on a notification - Fire open notification
```

Comment:
Changes themselves look good here to fix the inactive state.

Could you test opening a notification in the following states to ensure it still works.
* Tapping on a notification that cold starts the app. Swipe away the app, then tapping on a notification.
  - Possible this could be consider `UIApplicationStateInactive`
* Tapping on a notification while the app is in the foreground
* Tapping on a notification while the app is in the background

Reply:
Tapping on a notification that cold starts the app ends up being `UIApplicationStateActive` when this event fires